src/core/gui/TextEditor.cpp

```diff
@@ -353,55 +339,48 @@ void TextEditor::toggleOverwrite() {
  * I know it's a bit rough and duplicated
  * Improve that later on...
  */
-void TextEditor::decSize() {
+void TextEditor::decreaseFontSize() {
     XojFont& font = text->getFont();
-    double fontSize = font.getSize();
-    fontSize--;
-    font.setSize(fontSize);
-    setFont(font);
+    font.setSize(font.getSize() - 1);
```

Comment:
Does this need to check if the size > 1?

Reply:
Nice catch!